packages/e2e-test-utils/src/set-option.js

```diff
@@ -9,21 +9,42 @@ import { pressKeyWithModifier } from './press-key-with-modifier';
 /**
  * Sets a site option, from the options-general admin page.
  *
- * @param {string} setting The option, used to get the option by id.
- * @param {string} value   The value to set the option to.
+ * @param {string} setting   The option, used to get the option by id.
+ * @param {string} value     The value to set the option to.
+ * @param {string} adminPage The url of the admin page to visit.
+ *
+ * @return {Promise<string>} The current value of the option.
```

Comment:
Looks like we're no longer returning any value from the helper function?

Reply:
Yep

Reply:
Let's remove the JSDoc line it in case we don't actually return anything. I'd even lean towards removing it already so we don't forget about it; we can add it back later 🙂 YAGNI etc.